lib/urlwatch/reporters.py

```diff
@@ -96,8 +104,10 @@ def get_signature(self):
                                                       copyright=urlwatch.__copyright__),
             'Website: {url}'.format(url=urlwatch.__url__),
             'Support urlwatch development: https://github.com/sponsors/thp',
-            'watched {count} URLs in {duration} seconds'.format(count=len(self.job_states),
-                                                                duration=self.duration.seconds),
+            'watched {total} URLs in {duration} seconds'.format(
```

Comment:
Does it make sense to show here that something is filtered down? e.g. `watched 5 or 9 URLs in 12.34 seconds`?